### DISCUSSION TITLE
Standard Notes "auto-resolving" conflicts leads to duplicate notes

### DISCUSSION THREAD

**Discussion by ericswpark** [standardnotes/forum#2577] [General]
~~As per title, Standard Notes lost about 2-3 paragraphs of a note I was working on. Occurred on the Android app, version 3.14.0 installed from the Play Store.~~

**To Reproduce**
~~1. I wrote down about 4-5 paragraphs worth of text in a note file. I pressed the home button and navigated to a different app. I didn't come back to Standard Notes for about an hour.
2. When I came back to the Standard Notes app, it began to decrypt my notes and repopulate the list view. I clicked on the note I was previously working on.
3. The last 2-3 paragraphs of the 4-5 paragraphs I wrote on top disappeared.~~

**Expected behavior**
~~Standard Notes shouldn't lose my work~~

The above is incorrect; I did not lose my work. See comment after for details.

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy Note 20 Ultra
 - OS: Android 12
 - Version 3.14.0

**Additional context**
At step 2 of reproduction steps, I did see a brief red message on the note I was working on. Something along the lines of "Conflict detected". It went away before I could click on it. I'm guessing the way app decided to "auto-resolve" the conflict wiped out my work.

**Comment by ericswpark** [comment]
After further inspection it seems like the note I wrote previously is still there, just duplicated. The latest one (with my newest changes) existed below the old entry with an identical name, so I didn't notice. I've amended the title to reflect this.

However this is not ideal; Standard Notes should let me know that there is a conflict and ask me what I want to do about it, not silently create a second copy with no discernible features. Or at the very least it should show that the auto-resolved entry has been automatically resolved, maybe by appending the fact in the title.

**Comment by atmoio** [comment]
When it comes to conflict creation, we always keep the current note you're working on as it is, so that the text doesn't shift while you're editing. So whatever other version of the note is created as a "silent" copy. However, on that copy, you should see a red flag that says "Conflicted copy". Did this not appear?

**Comment by ericswpark** [comment]
@moughxyz  it appeared for half a second and then disappeared immediately, without me clicking anything. If it was meant to appear and stay, it might be a separate bug.
